components/servo/lib.rs

```diff
@@ -960,6 +963,9 @@ pub fn set_logger(script_to_constellation_chan: ScriptToConstellationChan) {
 
 /// Content process entry point.
 pub fn run_content_process(token: String) {
+    #[cfg(feature = "color_backtrace")]
+    color_backtrace::install();
```

Comment:
@jdm , question here 🙋‍♂️

IIUC, we just need to use this `#[cfg(feature = "color_backtrace")]` attribute and build with `--features color_backtrace`.

But, I currently still can't get this install run with `--features color_backtrace` :/ Did I miss something for using feature?

Thanks!

Reply:
Yes - the feature only applies to the top-level crate (ports/glutin). The feature needs to propagate to libservo with `libservo/color_backtrace` in that Cargo.toml.

Reply:
I see! Thanks!